### DISCUSSION TITLE
Restoring settings from backup and file-based configuration

### DISCUSSION THREAD

**Discussion by socketbox** [mozilla/multi-account-containers#2678] [Q&A]
I recently lost all of my container settings after "refreshing" firefox (it was crashing on AWS Console repeatedly).

Where can I find this configuration (container names, icons, colors) data? Is there a single file that can be used for configuration?

**Comment by kelimuttu** [comment]
Oh, no! Sorry about that. I'm not sure if it's possible. Looping in @dannycolin or @groovecoder who know better than me.

**Comment by H-Sapien** [comment]
Yes, this seems to be an important feature missing from MAC.
Surely a fundamental premise of information technology is to never burn your bridges behind you - until you're sure that any new revision is safe and effective?

I have an issue with the BitWarden extension in Firefox 137.0.2 in displaying a blank dropdown, so I want to remove my current extensions, including MAC, to see if I can isolate the culprit.
I would like to save my considerable MAC investment so that I can re-instate the settings after my BitWarden investigation, but where do we save/backup all our MAC data?

**Comment by nickelslots** [comment]
I wanted to jump in here to ask the same question.  I see someone has already done so. 
I suppose we don't have a way to do this, yet?
Would be really nice if we did.

**Comment by CPUGPU** [comment]
I would like a way to backup and restore the configuration, or transfer it to another computer.  Without it, I have to limit use to what I'm willing to manually recreate after a crash or on a different computer.

I tried to look for the data in the dotfiles, but where and how it's stored is beyond my skills.

**Comment by Javican** [comment]
Bumping this request to have a way to backup website assignments to each container

**Comment by H-Sapien** [comment]
Another bump.
Anyone heard of an IT system without a backup/export/import facility?

**Comment by icantremember** [comment]
bump... hard to believe the config is not easily exportable

**Comment by mforkel** [comment]
Yet another bump...
